create-ava-rule.js

```diff
@@ -129,16 +130,29 @@ module.exports = function createAvaRule() {
 		}
 	};
 
-	Object.defineProperty(rule, 'isTestFile', {
-		get: function () {
-			return isTestFile;
-		}
-	});
+	rule.isInTestFile = function () {
+		return isTestFile;
+	};
 
-	Object.defineProperty(rule, 'currentTestNode', {
-		get: function () {
-			return currentTestNode;
-		}
+	rule.isTestNode = function (node) {
+		return currentTestNode === node;
+	};
+
+	rule.isInTestNode = function () {
+		return currentTestNode;
+	};
+
+	rule.if = rest(function (predicates) {
```

Comment:
Yeah, colorwise, it gets shown with the same color as an if in m'y editor, and I rhoight that was actually pretty convenient, as that translates pretty well what it does. Happy to change it to when if you want..